.gitignore

```diff
@@ -1,4 +1,11 @@
+dist/
 .cabal-sandbox/
 cabal.sandbox.config
-dist/
-dist-newstyle/
+.stack-work/
+.ghc.environment.*
+/dist-newstyle
+/fuzz-temp-test.c
+/fuzz-temp-test.bc
+/fuzz-results
```

Comment:
What generates these? Worth leaving a comment.

Reply:
Oops, those were in the copy paste from llvm-pretty-bc-parser and aren't actually needed here.  Removed.